src/GameFace/Modules/ModuleAchievements.cs

```diff
@@ -0,0 +1,54 @@
+﻿using GameFace.Controllers;
+using System.Linq;
+using System.Threading.Tasks;
+using System;
+
+namespace GameFace.Modules
+{
+    public class ModuleAchievements: Nancy.NancyModule
```

Comment:
Do we want a way to add an achievement? If it is not part of the spec we can just script it in